eng/pipelines/coreclr/templates/build-test-job.yml

```diff
@@ -55,7 +64,11 @@ jobs:
     # because it needs System.Private.Corelib; we should be able to remove this dependency
     # by switching over to using reference assembly.
     ${{ if ne(parameters.stagedBuild, true) }}:
-      dependsOn: ${{ format('coreclr_product_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+      dependsOn:
+      - ${{ format('coreclr_product_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+      - ${{ if ne(parameters.liveLibrariesBuildConfig, '') }}:
+        - ${{ format('libraries_build_{0}_{1}{2}_{3}_{4}', 'netcoreapp', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveLibrariesBuildConfig) }}
```

Comment:
It might make sense to remove `netcoreapp` from the libraries build names when target framework is netcoreapp as libraries is the only subset that builds for other frameworks. That way we don't need to hard code it here. Something to consider on a follow up PR.